README.md

```diff
@@ -22,7 +22,7 @@ You can build and test the project using [Cargo](https://doc.rust-lang.org/cargo
 # Build the project
 cargo build
 
```

Comment:
Not sure if we want to break out redis into an optional feature. If not, then the redis test will fail if no local `redis-server` is installed